sqlgen/reflect.go

```diff
@@ -319,7 +332,7 @@ func makeWhere(table *Table, filter Filter) (*SimpleWhere, error) {
 			return nil, fmt.Errorf("unknown column %s", name)
 		}
 
-		l = append(l, whereElem{column: column, value: value})
+		l = append(l, whereElem{column: column, value: column.Descriptor.Valuer(reflect.ValueOf(value))})
```

Comment:
to verify - this is a departure from the existing behavior because the filters values are now converted vs. used directly.

Reply:
Yes